entrypoint.sh

```diff
@@ -79,8 +79,6 @@ if [ -z "$ORYX_DISABLE_TELEMETRY" ] || [ "$ORYX_DISABLE_TELEMETRY" == "false" ];
     export GITHUB_ACTIONS_BUILD_IMAGE_PULL_END_TIME=$endTime
 fi
 
-export ORYX_SDK_STORAGE_BASE_URL="https://oryxsdk-cdn.azureedge.net"
```

Comment:
Can you also remove the line 69 having `oryxCommand="${oryxCommand} --enable-dynamic-install"` since we no longer support this switch and also the github-actions build image has this environment variable to true by default

Reply:
yeah sure, thanks for reminding me.